### PR TITLE
i18n(zh-cn): update guides/integrations-guide/mdx.mdx

### PR DESCRIPTION
**Pull request by melon95**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
update guides/integrations-guide/mdx.mdx

### CODE REVIEW

**Comment by melon95** [comment]
@liruifengv

File: src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx [hunk 1]
```diff
@@ -179,6 +182,70 @@ export default defineConfig({
 
 我们建议[使用 AST Explorer](https://astexplorer.net/)来处理 estree 的输出，并尝试 [`estree-util-visit`](https://unifiedjs.com/explore/package/estree-util-visit/) 来搜索整个 JavaScript 节点。
 
+### 优化
+
+*   **类型：** `boolean | { customComponentNames?: string[] }`
+
+这是一个可选的配置设置，用于优化 MDX 输出，以便通过内部 rehype 插件加快构建和渲染速度。如果您的 MDX 文件较多，并注意到构建速度较慢，这可能会很有用。不过，该选项可能会生成一些未转义的 HTML，因此请确保您网站的交互式部分在启用该选项后仍能正常工作。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx:189]:
```suggestion
这是一个可选的配置设置，用于优化 MDX 输出，以便通过内部 rehype 插件加快构建和渲染速度。如果你的 MDX 文件较多，并注意到构建速度较慢，这可能会很有用。不过，该选项可能会生成一些未转义的 HTML，因此请确保你的网站的交互式部分在启用该选项后仍能正常工作。
```

File: src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx [hunk 2]
```diff
@@ -179,6 +182,70 @@ export default defineConfig({
...
+        // Prevent the optimizer from handling `h1` elements
+        // These will be treated as custom components
+        customComponentNames: ['h1'],
+      },
+    }),
+  ],
+});
+```
+
+请注意，如果你的 MDX 文件[使用 `export const components = {...}` 配置自定义组件](/zh-cn/guides/markdown-content/#assigning-custom-components-to-html-elements)，则您无需手动配置此选项。优化器将自动检测它们。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx:246]:
```suggestion
请注意，如果你的 MDX 文件[使用 `export const components = {...}` 配置自定义组件](/zh-cn/guides/markdown-content/#assigning-custom-components-to-html-elements)，则无需手动配置此选项。优化器将自动检测它们。
```

File: src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx [hunk 3]
```diff
@@ -179,6 +182,70 @@ export default defineConfig({
...
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  integrations: [
+    mdx({
+      optimize: {
+        // Prevent the optimizer from handling `h1` elements
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx:237]: 这里注释需要翻译

File: src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx [hunk 4]
```diff
@@ -179,6 +182,70 @@ export default defineConfig({
...
+    mdx({
+      optimize: true,
+    }),
+  ],
+});
+```
+
+#### customComponentNames
+
+*   **类型:** `string[]`
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx:209]:
```suggestion
*   **类型：** `string[]`
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx [hunk 5]
```diff
@@ -179,6 +182,70 @@ export default defineConfig({
...
+
+`optimize` 的一个可选属性，用于防止 MDX 优化器处理[通过组件属性传递给导入的 MDX 内容的任何自定义组件](/zh-cn/guides/markdown-content/#custom-components-with-imported-mdx)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/mdx.mdx:211]:
```suggestion
`optimize` 的一个可选属性，用于防止 MDX 优化器处理[通过组件属性传递给导入的 MDX 内容的任何自定义组件](/zh-cn/guides/markdown-content/#使用导入的-mdx-的自定义组件)。
```

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 💜

**Pull request merged**